docs/extras/integrations/providers/annoy.mdx

```diff
@@ -1,6 +1,6 @@
 # Annoy
 
-> [Annoy](https://github.com/spotify/annoy) (`Approximate Nearest Neighbors Oh Yeah`) is a C++ library with Python bindings to search for points in space that are close to a given query point. It also creates large read-only file-based data structures that are mmapped into memory so that many processes may share the same data. 
```

Comment:
this one is not a typo

Reply:
@obi1kenobi  You mean the only the extra 'm' in mmaped should be removed and 'that are' should still be 
 there,right?

Reply:
No, none of it is a typo. mmap is feature of modern operating systems: https://man7.org/linux/man-pages/man2/mmap.2.html

Reply:
Understood, Thanks @obi1kenobi , I will revert this change in my PR